src/popup/components/Modals/ConfirmTransactionSign.vue

```diff
@@ -516,6 +563,19 @@ export default defineComponent({
     height: 56px;
   }
 
+  .subtitle {
+    margin: 8px 0;
+    color: variables.$color-grey-light;
+
+    &.error {
+      color: variables.$color-warning;
```

Comment:
The class name does not match the color used. I suggest to use `warning` class name.